### PR TITLE
roles/{consul,dev}: add prebake tag & use in dev

### PR DESCRIPTION
**Pull request by unclejack**
This fixes consul to make it available for the dev role as well.

### CODE REVIEW

File: roles/dev/meta/main.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependencies:
 - { role: ansible, tags: 'prebake-for-dev' }
 - { role: docker }
 - { role: etcd }
+- { role: consul }
```
Comment (gaurav-dalvi) [inline roles/dev/meta/main.yml:18]: why are we adding both consul and etcd roles ?

Comment (unclejack) [reply roles/dev/meta/main.yml:18]: This might need a bit more work. However, we need consul in the Vagrant boxes so that the netplugin tests can use it.

**Comment by unclejack** [comment]
This PR addresses the issue described in this comment: https://github.com/contiv/netplugin/pull/609#issuecomment-260543228

This PR will enable us to build new boxes and switch to those.

This PR is ready for review.

**Review by dseevr — APPROVED** [review]

**Comment by gaurav-dalvi** [comment]
LGTM

**Pull request merged**